fluidBuild.config.cjs

```diff
@@ -23,7 +23,14 @@ module.exports = {
 			script: false,
 		},
 		"compile": {
-			dependsOn: ["commonjs", "build:esnext", "build:test", "build:copy"],
+			dependsOn: [
+				"commonjs",
+				"build:esnext",
+				"build:test",
+				"build:copy",
+				// "^build:rename-types",
```

Comment:
To be enabled later? Should this be removed?

Reply:
Yeah, I'll fix this. Left it for the first of my many PRs to get approved/merged. :)